homeassistant/helpers/config_validation.py

```diff
@@ -758,7 +758,7 @@ def ensure_list_csv(value: Any) -> list:
 class multi_select:
     """Multi select validator returning list of selected values."""
 
-    def __init__(self, options: dict) -> None:
+    def __init__(self, options: dict | list) -> None:
```

Comment:
Currently, `config_validation.multi_select` accepts only a `dict`, and then returns a `list`.

I couldn't think of a reason to not enable it for lists also.

Reply:
This is good.